src/x/instrument/config.go

```diff
@@ -107,6 +107,23 @@ type NewRootScopeAndReportersOptions struct {
 	CommonLabels map[string]string
 }
 
+type metricsClosers struct {
+	serverCloser   io.Closer
```

Comment:
can you add some high level comments for what differentiates these closers?

Reply:
Good call. Updated!